### PR TITLE
FIx #80 and  fix compatibility with python2.7

### LINKED ISSUE #80: CI did not report the failed tests
image

I found some unexpected errors, it looks contain some failures with python2.7.

Because I override this function:

```python
    def vsParse(self, sbytes, offset=0, fast=False):
        self.sbytes = sbytes
        self.len_sbytes = len(sbytes)
        return super().vsParse(sbytes, offset, fast)
```

And the test command `find . -name test\*py -exec py.test {} -v --cov=idb \;`, did not report them(the failed tests).

_Originally posted by @imbillow in https://github.com/williballenthin/python-idb/pull/79#issuecomment-658875999_

### PR DESCRIPTION
**Pull request by imbillow**

### CODE REVIEW

**Comment by imbillow** [comment]
I also detected the single test method consumes 11GB memory, when running the tests with python2.7. Amazing!

```
@slow
@kern32_test()
def test_cursor_enum_all_asc(kernel32_idb, version, bitness, expected):
    minkey = kernel32_idb.id0.get_min().key
    cursor = kernel32_idb.id0.find(minkey)
    count = 1
    while True:
        try:
            cursor.next()
        except IndexError:
            break
        count += 1

    assert kernel32_idb.id0.record_count == count
```

Figure_test_idb_test_cursor_enum_all_asc_py27

**Comment by XVilka** [comment]
@williballenthin by the way, it probably makes sense to carve a new release after this one is merged.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -33,5 +33,9 @@ jobs:
       - name: Run tests
         shell: bash
         run: |
+          set -e
           find . -name \*.py -exec pycodestyle --ignore=E501 {} \;
-          find . -name test\*py -exec py.test {} -v --cov=idb \;
+          for f in $(find . -name test\*py)
+          do
+            py.test -v --cov=idb $f
+          done
```
Comment (williballenthin) [inline .github/workflows/main.yml:41]: good idea

**Comment by williballenthin** [comment]
https://github.com/williballenthin/python-idb/releases/tag/v0.6.0

**Comment by XVilka** [comment]
@williballenthin perfect, thanks! Could you please to upload it to PyPI too? https://pypi.org/project/python-idb/ shows old 0.5.0 version.

**Comment by williballenthin** [comment]
whoops, thought we had an action to auto-push to pypi. will set this up now

**Comment by williballenthin** [comment]
https://pypi.org/project/python-idb/0.6.0/

**Pull request merged**